File: Conditionals/time.py
# Lisa Rehm, Time of Day Python

import time 

current =time.time()
local_time = time.localtime(current)
minute = local_time.tm_min
hour = local_time.tm_hour
print(hour)


if hour <= 5:
    print("Go to bed... please!!\n")
elif hour <= 11:
    print("Good Morning, are you well rested?!\n")
elif hour == 12:
    print("Good day! I'm giving you one hour to find your shadow.\n")
elif hour <= 16:
    print("Good afternoon! Don't go outside if it's summer, it's way too hot.\n")
elif hour <= 19:
    print("Good evening! Now you can go outside, have fun!\n")
elif hour <= 23:
    print("Good night! Please go to bed at a reasonable hour.\n")
else:
    print("That is not a time on this planet...\n")

File: Conditionals/time.c
//Lisa Rehm, Time of Day C
#include <stdio.h>
#include <time.h>


int main(void){
    time_t now = time(NULL);
    struct tm * tm_struct = localtime(&now);
    int hour = tm_struct->tm_hour;
    printf("%d\n", hour);

    if (hour <= 7){
        printf("Good morning! Tis the best time of day!.\n");
    }else if(hour <= 11){
        printf("Good near end of waking period, you should start preparing for sleep now.\n");
    }else if(hour == 12){
        printf("Good day! You should probably fall asleep now before the sun gets too bright.\n");
    }else if(hour <= 16){
        printf("Good afternoon! Enjoy your sleep!\n");
    }else if(hour <= 19){
        printf("Good evening! I hope you are well rested!\n");
    }else if(hour <= 23){
        printf("Good night, Enjoy your day!\n");
    }else{
        printf("Go back to the non-24-hour planet from which you came. Or stay. I don't care.\n");
    }
    return 0;
}
